Common/tests/Helpers/TestPollingAgentProvider.cs

```diff
@@ -126,28 +123,28 @@ public TestPollingAgentProvider(IWorkerStreamHandler workerStreamHandler)
     var computePlanOptions = builder.Configuration.GetRequiredValue<ComputePlane>(ComputePlane.SettingSection);
     builder.Services.AddSingleton(computePlanOptions);
 
-    app = builder.Build();
+    app_ = builder.Build();
 
-    resultTable_  = app.Services.GetRequiredService<IResultTable>();
-    taskTable_    = app.Services.GetRequiredService<ITaskTable>();
-    sessionTable_ = app.Services.GetRequiredService<ISessionTable>();
-    Submitter     = app.Services.GetRequiredService<ISubmitter>();
-    pollster_     = app.Services.GetRequiredService<Common.Pollster.Pollster>();
+    app_.Services.GetRequiredService<IResultTable>();
+    app_.Services.GetRequiredService<ITaskTable>();
```

Comment:
You don't need to call `GetRequiredService` if you don't do anything with its result.